docs/UserGuide.md

```diff
@@ -359,3 +383,15 @@ Action | Format, Examples
 **Clear** | `clear`
 **Help** | `help`
 **Exit** | `exit`
+
+---------------------------------------------------------------------------------------------------------------------
+
+## Glossary
+
+\# |            Term                | Description
+---|--------------------------------|------------------------------------------------------------------------------|
+1  | Command Line Interface (CLI)   | A text-based user interface (UI) used to view and manage computer files.     |
+2  | Graphical User Interface (GUI) | A system of interactive visual components for computer software.             |
+3  | Deliverable                    | Refers to items which need to be completed by a specified time.              |
+4  | Stakeholder                    | An external party involved with the product.                                 |
+5  | Milestone                      | Refers to a transition when a specified group of deliverables are completed. |
```

Comment:
I think this can be clearer, maybe something like A stage in the software development process associated with a particular group of deliverables.